sdcm/utils/distro.py

```diff
@@ -149,6 +150,10 @@ def is_ubuntu18(self):
     def is_ubuntu20(self):
         return self == self.UBUNTU20
 
+    @property
+    def is_ubuntu21(self):
+        return self == self.UBUNTU21
```

Comment:
If we add 21.10 in addition to 21.04 then this one must consdier both versions too.

Reply:
yeah, you got a point here.

it was weird UBUNTU21 was added without this function...

Reply:
fixed